test/regress/regress0/options/ast-and-sexpr.smt2

```diff
@@ -0,0 +1,10 @@
+; EXPECT: (SEXPR (SEXPR check-sat (CONST_RATIONAL 2)) (SEXPR * (CONST_RATIONAL 1)))
+; EXPECT: (:status unknown)
+
+(set-option :output-language ast)
+(set-option :command-verbosity (* 1))
```

Comment:
Can you comment in the benchmark (with `;`) what these commands mean?

Reply:
Done!